canonical-web-and-design/juju.is
# Update vanilla and fix tutorial layout

## Done
- Bumped Vanilla version
- Fixed the layout of tutorials

## QA
- Go to https://juju-is-248.demos.haus/tutorials/how-to-write-a-tutorial#1-overview
- Check that the numbers in the side nav are in circles

## Issue
Fixes #243
